meteor/client/ui/RundownView/RundownTiming.tsx

```diff
@@ -878,6 +1023,8 @@ export const CurrentPartRemaining = withTiming<IPartRemainingProps, {}>({
 
 interface ISegmentDurationProps {
 	partIds: PartId[]
+	budgetDuration?: number
+	livePosition: number
```

Comment:
I'd rename `livePostion` to something like `playedOutDuration`. We support `RundownPlaylist.outOfOrderTiming` which allows to treat unplayed parts as "still to be played" and not "skipped forever". Therefore, whether current `livePostion` means that these "past", unplayed parts will never be played (and therefore can be considered played as far as the budget goes) or not, is dependent on `outOfOrderTiming` value. Therefore, I'd change this to `playedOutDuration` so that user components can make the decision themselves and not to confuse this with SegmentComponent's `livePosition`, which is mostly a display property and I feel is related to timing only "coincidentally".